### PR TITLE
Automatic port of System docs (Convert, Decimal, Environment)

### PR DESCRIPTION
**Pull request by carlossanlop**

### CODE REVIEW

File: xml/System/Convert.xml [hunk 1]
```diff
@@ -906,10 +908,14 @@
...
       <Docs>
-        <param name="s">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="s">The string to convert.</param>
+        <summary>Converts the specified string, which encodes binary data as hex characters, to an equivalent 8-bit unsigned integer array.</summary>
+        <returns>An array of 8-bit unsigned integers that is equivalent to <paramref name="s" />.</returns>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.ArgumentNullException">
+          <paramref name="s" /> is <code>null</code>.</exception>
```
Comment (pgovind) [inline xml/System/Convert.xml:916]:
```suggestion
          <paramref name="s" /> is <see langword="null" />.</exception>
```

File: xml/System/Convert.xml [hunk 2]
```diff
@@ -8163,10 +8169,14 @@
...
       <Docs>
-        <param name="inArray">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="inArray">An array of 8-bit unsigned integers.</param>
+        <summary>Converts an array of 8-bit unsigned integers to its equivalent string representation that is encoded with uppercase hex characters.</summary>
+        <returns>The string representation in hex of the elements in <paramref name="inArray" />.</returns>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.ArgumentNullException">
+          <paramref name="inArray" /> is <code>null</code>.</exception>
```
Comment (carlossanlop) [inline xml/System/Convert.xml:8177]:
```suggestion
          <paramref name="inArray" /> is <see langword="null" />.</exception>
```

File: xml/System/Convert.xml [hunk 3]
```diff
@@ -8233,12 +8245,21 @@
...
-        <returns>To be added.</returns>
+        <param name="inArray">An array of 8-bit unsigned integers.</param>
+        <param name="offset">An offset in <paramref name="inArray" />.</param>
+        <param name="length">The number of elements of <paramref name="inArray" /> to convert.</param>
+        <summary>Converts a subset of an array of 8-bit unsigned integers to its equivalent string representation that is encoded with uppercase hex characters.
+            Parameters specify the subset as an offset in the input array and the number of elements in the array to convert.</summary>
+        <returns>The string representation in hex of <paramref name="length" /> elements of <paramref name="inArray" />, starting at position <paramref name="offset" />.</returns>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.ArgumentNullException">
+          <paramref name="inArray" /> is <code>null</code>.</exception>
```
Comment (carlossanlop) [inline xml/System/Convert.xml:8256]:
```suggestion
          <paramref name="inArray" /> is <see langword="null" />.</exception>
```

File: xml/System/Environment.xml [hunk 4]
```diff
@@ -1663,7 +1663,7 @@ To identify the operating system platform, for example, Linux or Windows, you ca
         <ReturnType>System.Int32</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets the unique identifier for the current process.</summary>
         <value>To be added.</value>
```
Comment (pgovind) [inline xml/System/Environment.xml:1667]: Does value need to be documented?

Comment (carlossanlop) [reply xml/System/Environment.xml:1667]:
```suggestion
        <value>A number that represents the Process Id.</value>
```

**Review by pgovind — APPROVED** [review]

**Pull request merged**